tools/accuracy_checker/dataset_definitions.yml

```
@@ -752,3 +752,12 @@ datasets:
       - type: ssim
         presenter: print_vector
     annotation: style_trasfer_val2017.pickle
+
+  - name: squad_v1_1_msl384_mql64_ds128_lowercase
+    annotation_conversion:
+      converter: squad
+      testing_file: squad_v1.1/dev-v1.1.json
+      max_seq_length: 384
+      max_query_length: 64
+      doc_stride: 128
+      lower_case: true
```

please fix yamllint comment:
```
11:29:36    763:23    error    no new line character at the end of file  (new-line-at-end-of-file)
```
```suggestion
      lower_case: true

```

---

P.S. it is the last comment from my side,  I hope :)
@eizamaliev do you have something to say?